.github/workflows/pr-agent.yml

```
@@ -39,5 +40,5 @@ jobs:
         uses: The-PR-Agent/pr-agent@v0.36.1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          GOOGLE_AI_STUDIO.GEMINI_API_KEY: ${{ secrets.GEMINI_TOKEN }}
+          GOOGLE_AI_STUDIO.GEMINI_API_KEY: ${{ secrets.GEMINI }}
```

actionlint (1.7.12)

[error] property "gemini" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; gemini_token: string; github_token: string}

(expression)